src/subcategories/other.ts

```diff
@@ -0,0 +1,13 @@
+import { makeEnum } from '@transcend-io/type-utils';
+
+/** Information that does not belong to an individual */
+export const OtherDataSubCategory = makeEnum({
```

Comment:
G I'm not sure this is right - `Other` is the fallback sub category for each category, so I don't know if it makes sense to add this

Reply:
Yeah, that's my worry too. Would love to hear Alessandro feedback here as well because I wonder if he saw something like this before for the classifier

Reply:
> @MadDataScience wdyt?

@giacaglia this doesn't seem right. I see the model at s3://dev-py-datapoint-classifier-resources/content_classification/samp_OTHER_v13.joblib I'm not sure how it was created. There should be no model for "Other". Rather, there is one for each parent category. 

I'm wondering if a customer labeled something "Other". If something got labeled "Other" that could either be because a customer added an "Other" category (@anotherminh can you verify if this is possible?) or because of a glitch where one of the "Other" subcategories was not interpreted as its parent when it should have been. If it's the former, we might want to put some guard in place to make sure this doesn't happen in the future. If the latter, well, we need to fix that bug.

Reply:
Perfect!

Reply:
I will take a look at this!